python/Dockerfile

```diff
@@ -201,6 +201,6 @@ COPY --from=rust /usr/local/cargo $CARGO_HOME
 # Configure cargo to use Git CLI so the Git shim works
 RUN mkdir -p ~/.cargo && printf "[net]\ngit-fetch-with-cli = true\n" >> ~/.cargo/config.toml
 
-COPY --chown=dependabot:dependabot cargo $DEPENDABOT_HOME/cargo
-COPY --chown=dependabot:dependabot common $DEPENDABOT_HOME/common
+COPY --chown=dependabot:dependabot --parents cargo common $DEPENDABOT_HOME/
```

Comment:
these two could be combined, but I didn't want to do that in the scope of this PR... as there's a bit more that could actually be combined...